cmd/kube-controller-manager/app/core.go

```diff
@@ -324,21 +324,22 @@ func startResourceQuotaController(ctx ControllerContext) (http.Handler, bool, er
 }
 
 func startNamespaceController(ctx ControllerContext) (http.Handler, bool, error) {
+	namespaceKubeClient := ctx.ClientBuilder.ClientOrDie("namespace-controller")
```

Comment:
Do we really want to construct namespaceKubeClient twice in different ways?

Reply:
Fixed.